tests/bootstrap.php

```diff
@@ -9,6 +9,6 @@
  * file that was distributed with this source code.
  */
 
-$loader = require_once __DIR__ . "/../vendor/autoload.php";
+$loader = require __DIR__ . "/../vendor/autoload.php";
```

Comment:
Ok. I've taken this out of this pull request as I'm not convinced that is correct. We can address it separately if it continues to be problematic.